### ISSUE TITLE
Broken on frontpage?

### ISSUE THREAD

**Issue by jerbob92** [prerender/prerender#380]
I have an Angular website and I currently have the issue that Prerender is working on every page, except for the frontpage.
When I visit another route (/home) which results in the exact same HTML, it works.
I can see that Prerender is doing the Ajax calls, but it's not doing everything it should (like downloading images). It looks like it's hanging somewhere, but I have no idea how to figure out where.

And it's really strange that it's only happening on the frontpage.

I see the following in the logs of Prerender:
```
prerender_1     | 2017-01-29T18:50:33.691Z getting http://nginx_static:7777/
prerender_1     | 2017-01-29T18:50:45.735Z Timed out. Sending request with HTML on the page
prerender_1     | 2017-01-29T18:50:45.753Z got 200 in 12062ms for http://nginx_static:7777/
```

**Comment by thoop** [comment]
Hmm, that is weird. Can you send your URL to us at support@prerender.io? That way we can look into by testing your URL with a local Prerender server.

**Comment by jerbob92** [comment]
Thanks, I have sent you an email. Is there any way I can turn on some verbose logging? (not just the normal logger plugin)

**Comment by jerbob92** [comment]
I have found the issue. It had to do with CORS headers and caching. It was just coincidence that it didn't work on the frontpage.
I cached the API calls in Drupal, which also included the "Access-Control-Allow-Origin" header, the value for this was wrong as the origin is different because I route the Prerender fetching internally in Docker.

I have set the "Access-Control-Allow-Origin" to "*" now until I find a better solution.
